gap/PGMatrix.gi

```diff
@@ -1,56 +1,89 @@
-# Type PGMatrixElements
-DeclareRepresentation("IsPGMatrixElementsComponentRep", IsComponentObjectRep,
-	[ "signature", "quotient", "sparse", "elements" ]
+# Type PGMatricesOfGenerators
+DeclareRepresentation("IsPGMatricesOfGeneratorsComponentRep", IsComponentObjectRep,
+	[ "fulltg", "tg", "tgquotient", "sparse", "GeneratorNames", "PGMatricesRec" ]
 );
 
 
 InstallMethod( \=, [
-    IsPGMatrixElementsObj and IsPGMatrixElementsComponentRep,
-    IsPGMatrixElementsObj and IsPGMatrixElementsComponentRep ], 
-function(pgMatElements1, pgMatElements2)
-	return  Signature(pgMatElements1) = Signature(pgMatElements2) and
-		TGQuotientName(pgMatElements1) = TGQuotientName(pgMatElements2) and
-		GetElements(pgMatElements1) = GetElements(pgMatElements2);
+    IsPGMatricesOfGeneratorsObj and IsPGMatricesOfGeneratorsComponentRep,
+    IsPGMatricesOfGeneratorsObj and IsPGMatricesOfGeneratorsComponentRep ], 
+function(pgMatsGs1, pgMatsGs2)
+	return  GetTriangleGroup(pgMatsGs1) = GetTriangleGroup(pgMatsGs2) and
+		GetProperTriangleGroup(pgMatsGs1) = GetProperTriangleGroup(pgMatsGs2) and
+		GetTGQuotient(pgMatsGs1) = GetTGQuotient(pgMatsGs2) and
+		PGMatricesRec(pgMatsGs1) = PGMatricesRec(pgMatsGs2);
 end );
 
 
-InstallMethod( Signature, [ IsPGMatrixElementsObj and IsPGMatrixElementsComponentRep ],
-function(pgMatElements)
-	return pgMatElements!.signature;
+# This should only be used internally for the test files !!!
```

Comment:
If this should only be used in test files, maybe it would be appropriate to define it there locally?